cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/velocity/VelocityCloudNetBridgePlugin.java

```diff
@@ -73,13 +73,16 @@ private void initListeners() {
 
         //CloudNet
         CloudNetDriver.getInstance().getEventManager().registerListener(new VelocityCloudNetListener());
-        CloudNetDriver.getInstance().getEventManager().registerListener(new VelocityPlayerExecutorListener(this.proxyServer));
```

Comment:
Why did you remove this?

Reply:
I didn't even noticed
Well, I started editing the master branch, then switch to development. Now it should be clear why some stuff is missing, like the `this` accessor.
Sorry, I'll fix that :/